homeassistant/components/cloud/http_api.py

```diff
@@ -172,6 +173,29 @@ def post(self, request, data):
         return self.json_message('ok')
 
 
+class CloudResendConfirmView(HomeAssistantView):
+    """Resend email confirmation code."""
+
+    url = '/api/cloud/resend_confirm'
+    name = 'api:cloud:resend_confirm'
+
+    @_handle_cloud_errors
+    @RequestDataValidator(vol.Schema({
+        vol.Required('email'): str,
+    }))
+    @asyncio.coroutine
+    def post(self, request, data):
+        """Handle resending confirm email code request."""
+        hass = request.app['hass']
+        cloud = hass.data[DOMAIN]
+
+        with async_timeout.timeout(REQUEST_TIMEOUT, loop=hass.loop):
+            yield from hass.async_add_job(
```

Comment:
Isn't `async_add_job` a callback nowadays? So we don't yield from that, right?

Reply:
It's a callback that returns a future when you add a sync function. This way you can wait till the sync function is done.

Reply:
Gotcha, thanks!